contracts/seed/Seed.sol

```diff
@@ -712,6 +709,36 @@ contract Seed {
         }
     }
 
+    function getAllClasses()
+        external
+        view
+        returns (
+            bytes32[] memory classNames,
+            uint256[] memory classCaps,
+            uint256[] memory individualCaps,
+            uint256[] memory vestingCliffs,
+            uint256[] memory vestingDurations,
+            uint256[] memory classFundingsCollected
+        )
+    {
+        uint256 numberOfClasses = classes.length;
+        classNames = new bytes32[](numberOfClasses);
+        classCaps = new uint256[](numberOfClasses);
+        individualCaps = new uint256[](numberOfClasses);
+        vestingCliffs = new uint256[](numberOfClasses);
+        vestingDurations = new uint256[](numberOfClasses);
+        classFundingsCollected = new uint256[](numberOfClasses);
+        for (uint256 i; i < numberOfClasses; ++i) {
+            ContributorClass storage class = classes[i];
+            classNames[i] = class.className;
+            classCaps[i] = class.classCap;
+            individualCaps[i] = class.individualCap;
+            vestingCliffs[i] = class.vestingCliff;
+            vestingDurations[i] = class.vestingDuration;
+            classFundingsCollected[i] = class.classFundingCollected;
+        }
+    }
```

Comment:
missing return statement, or solidity works like this?

Reply:
so the return statement can be defined in the function prototype, where we define the return types. At the end of the function, it will return the variables automatically